### DISCUSSION TITLE
OSM tiles alternative?

### DISCUSSION THREAD

**Discussion by poutineflyer** [albar965/littlenavmap#841] [General]
Hello,
it seems that tiles.wmflabs.org to have OSM tiles is gonna be discontinued any plan to replace that?
Thank you!

**Comment by albar965** [comment]
There is no replacement for this (use OpenTopoMap or StamenTerrain for now) but I added options for more maps like Thunderforest or Mapbox for the next release.

The hillshading was buggy and limited anyway.
Alex
